src/main/java/com/courtcanva/ccfranchise/config/SecurityConfig.java

```diff
@@ -0,0 +1,44 @@
+package com.courtcanva.ccfranchise.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.web.cors.CorsConfiguration;
+
+import java.util.List;
+
+
+@Configuration
+@EnableWebSecurity
+@ConfigurationProperties(prefix = "management.endpoints.web.cors")
+public class SecurityConfig {
+    @Value("${allowed-origins}")
+    private List<String> origins;
+
+    @Value("${allowed-methods}")
+    private List<String> methods;
+
+    @Value("${allowed-headers}")
+    private List<String> headers;
+
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
```

Comment:
This `filterChain` is a new way introduced in the latest version's Spring Security to do the configuration, right? Good job!